csm/core/services/health.py

```diff
@@ -506,11 +521,11 @@ def update_health_map(self, msg_body):
                     resource_schema_dict[const.ALERT_UUID] \
                         = msg_body.get(const.ALERT_UUID, "NA")
                     resource_schema_dict[const.FETCH_TIME] \
-                        = msg_body.get(const.FETCH_TIME, "")
+                        = msg_body.get(const.FETCH_TIME, "NA")
```

Comment:
What is fetch time and why it should be empty or NA in some cases?

Reply:
Fetch time is the event_time . It will always be there. Corrected the code.